docs/getting_started.md

```diff
@@ -95,6 +99,25 @@ spec:
 
 Both labels and annotations can be added to the Prometheus configuration, so that metrics are associated with the appropriate information, such as the test and tenant ids. You can read more about this in the [Prometheus documentation](https://prometheus.io/docs/prometheus/latest/configuration/configuration/#kubernetes_sd_config) site.
 
+##### Image pull policy and secrets
+
+You can use a private image for master and worker images, and configure its pull policy and corresponding secret. For example:
```

Comment:
Adding more clarification since you can already use (private) images from any registry for as long as authentication is not required on a pod level through secrets e.g. if cluster nodes are configured already by the cloud admin.  

Proposal:
```suggestion
Images from private image registry can be used through various methods as described in [kubernetes documentation here](https://kubernetes.io/docs/concepts/containers/images/#using-a-private-registry), one of those methods depends on setting `ImagePullSecrets` for pods. This is supported by simply setting the `imagePullSecrets` in the deployed custom resource. 
<EXAMPLE CODE SNIPPET>
```

I am leaving out `imagePullPolicy` since I believe it should have its own (although small) sub-section.